recipes/glib/all/test_package/conanfile.py

```diff
@@ -16,33 +15,6 @@ def layout(self):
     def requirements(self):
         self.requires(self.tested_reference_str)
 
-    def build_requirements(self):
-        if self.settings.os != "Windows" and not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
-            self.tool_requires("pkgconf/2.0.3")
-
-    def generate(self):
-        tc = CMakeToolchain(self)
-        tc.generate()
-        virtual_run_env = VirtualRunEnv(self)
-        virtual_run_env.generate()
-
-        if self.settings.os == "Macos":
-            env = Environment()
-            # Avoid conflicts with system libiconv
-            # see: https://github.com/conan-io/conan-center-index/pull/17610#issuecomment-1552921286
-            env.define_path("DYLD_FALLBACK_LIBRARY_PATH", "$DYLD_LIBRARY_PATH")
-            env.define_path("DYLD_LIBRARY_PATH", "")
-            env.vars(self, scope="run").save_script("conanrun_macos_runtimepath")
```

Comment:
it looks like this is still needed, cf https://github.com/ericLemanissier/proof-of-conan/actions/runs/9177037237/job/25233751761#step:8:2039